torch/onnx/operators.py

```diff
@@ -0,0 +1,24 @@
+import torch
+from torch.autograd import Variable
+import torch.onnx
+
+# This file contains pure pytorch operators that should eventually
+# find a home outside of the onnx/ directory
```

Comment:
I don't think we'll want to ever move this elsewhere. I think the plan for supporting functions that depend on shapes isn't tracing, but JIT script

Reply:
ok, i'll take out the comment